packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -366,6 +366,107 @@ export = {
       },
     },
 
+    'with client\'s TLS certificate from SDS': {
+      'should add a backend default to the resource with TLS certificate'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
+        // WHEN
+        new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          serviceDiscovery: appmesh.ServiceDiscovery.dns('test'),
+          backendDefaults: {
+            tlsClientPolicy: {
+              certificate: appmesh.TlsCertificate.sds( {
+                secretName: 'secret_certificate',
+              }),
+              ports: [8080, 8081],
+              validation: {
+                subjectAlternativeNames: {
+                  exactMatch: ['mesh-endpoint.apps.local'],
+                },
+                trust: appmesh.TlsValidationTrust.sds({
+                  secretName: 'secret_validation',
+                }),
+              },
+            },
+          },
+        });
+
+        // THEN
+        expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualNode', {
+          Spec: {
+            BackendDefaults: {
+              ClientPolicy: {
+                TLS: {
+                  Certificate: {
+                    SDS: {
+                      SecretName: 'secret_certificate',
+                    },
+                  },
+                  Ports: [8080, 8081],
+                  Validation: {
+                    SubjectAlternativeNames: {
+                      Match: {
+                        Exact: ['mesh-endpoint.apps.local'],
+                      },
+                    },
+                    Trust: {
+                      SDS: {
+                        SecretName: 'secret_validation',
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        }));
+
+        test.done();
+      },
+    },
+
+    'with client\'s TLS certificate from ACM': {
```

Comment:
Let's just use double quotes for strings that include `'`:

```suggestion
    "with client's TLS certificate from ACM": {
```

Reply:
Addressed.